phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/obo/hpo/HpoAnnotationLine.java

```diff
@@ -236,7 +237,8 @@ static HpoAnnotation toHpoAnnotation(HpoAnnotationLine line, Ontology ontology)
       frequencyString,
       onset,
       getModifiers(line.getModifierList()),
-      line.getPublication());
+      line.getPublication(),
+      EvidenceCode.fromString(line.evidence));
```

Comment:
Have you tested what happens here is evidence is null?

Reply:
fixed this.